client/js/store/modules/game.js

```diff
@@ -59,7 +59,7 @@ function handleGameResponse(context, jwres, resolve, reject) {
  * @param player is the player object
  */
 function queenCount(player) {
-  if (!player) {
+  if (!player || !player.faceCards) {
```

Comment:
Made a few of these getters more defensive. Vue was throwing an error inside the getter that wasn't breaking anything but was making dev tools sad.